pretty_gpx/common/data/place_name.py

```diff
@@ -47,9 +47,9 @@ def get_start_end_named_points(gpx_track: GpxTrack | MultiGpxTrack) -> list[Scat
                          lon=start_lon,
                          lat=start_lat,
                          category=ScatterPointCategory.START)
-    end = ScatterPoint(name=None,
-                       lon=start_lon,
-                       lat=start_lat,
+    end = ScatterPoint(name=get_place_name(lon=end_lon, lat=end_lat),
+                       lon=end_lon,
+                       lat=end_lat,
                        category=ScatterPointCategory.END)
```

Comment:
Really nice track @FlorianVeaux by the way :) 
How long did it take you to complete?

Reply:
Fixed